### PR TITLE
Aurelia JS framework detection

### PR DESCRIPTION
**Pull request by jhewt**
This adds a simple detection for the Aurelia, a Javascript Framework.
Since developers may use many different build tools the only way to detect it is to read attributes.

### CODE REVIEW

**Comment by jhewt** [comment]
A site with a list of sites built with Aurelia to test the rules: http://builtwithaurelia.com/

File: src/apps.json [hunk 1]
```diff
@@ -709,6 +709,21 @@
...
+		"Aurelia": {
+			"cats": [
+				12
+			],
+			"icon": "Aurelia.svg",
+			"html": [
+				"aurelia-app=[^>]",
+				"data-main=[^>]aurelia-bootstrapper",
+				"au-target-id=[^>]\\d"
+			],
```
Comment (AliasIO) [inline src/apps.json]: The HTML patterns are too liberal and match plaintext. Please change it to something that contains a HTML opening tag, e.g. `<[^>]+aurelia-app=`.

Comment (jhewt) [inline src/apps.json]: Indeed, fixed now. Thanks!

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**